packages/ingest/test/test-ftp.js

```diff
@@ -52,11 +56,12 @@ test('useList defaults to false when not assigned', async (t) => {
   t.is(jsftpSpy.getCall(0).args[0].useList, false);
 });
 
-test('Download remote file to s3', async (t) => {
+test('Download remote file to s3 with correct content-type', async (t) => {
   class MyTestFtpDiscoveryClass extends TestFtpMixin(MyTestDiscoveryClass) {}
   const myTestFtpDiscoveryClass = new MyTestFtpDiscoveryClass();
   const bucket = randomString();
   const key = randomString();
+  const expectedContentType = 'binary/octet';
```

Comment:
So right now we expect `binary/octet` because we expect `mime.lookup` for an `.hdf` file to return `null`?

Reply:
Yes to everything except we expect `mime.lookup` for a `.hdf` file to return `false`.

Reply:
nvm, I need to look into this.

Reply:
We currently expect a `'binary/octet'` because `const key = randomString();`. This means that we're checking against a string that doesn't have our expected extension. I'm going to change this.